instaloader/structures.py

```diff
@@ -43,18 +43,68 @@ class PostCommentAnswer(NamedTuple):
 PostCommentAnswer.likes_count.__doc__ = "Number of likes on comment."
 
 
-class PostComment(NamedTuple):
-    id: int
-    created_at_utc: datetime
-    text: str
-    owner: 'Profile'
-    likes_count: int
-    answers: Iterator[PostCommentAnswer]
+class PostComment:
+    def __init__(self, node: Dict[str, Any], answers: Iterator['PostCommentAnswer'],
+                 context: 'InstaloaderContext', shortcode: str):
```

Comment:
The order of the parameters in the constructor for classes like these is always `context`, `node`, and then additional parameters.
```suggestion
    def __init__(self, context: 'InstaloaderContext', node: Dict[str, Any],
                 answers: Iterator['PostCommentAnswer'], shortcode: str):
```